src/chat/zulip.md

```diff
@@ -0,0 +1,84 @@
+## Zulip
+
+[Rust's Zulip](rust-lang.zulipchat.com) is used by a number of teams, notably the compiler and language
+teams, along with their working groups.
+
+Zulip can be an unintuitive platform to get started with. The UI is divided into
+two areas: a list of streams on the left, and the main chat in the middle.
```

Comment:
Is there usage documentation of Zulip itself that we can point to?

Reply:
Yes, and it's great, maybe add something like ...

```suggestion
two areas: a list of streams on the left, and the main chat in the middle. Check out the [Zulip user documentation](https://zulipchat.com/help/) to learn more about Zulip.
```